src/styles/product-card.css

```
@@ -0,0 +1,93 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  max-width: 166px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-height: 358px;
+}
+
+.product-card__img {
+  display: flex;
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  object-fit: contain;
+  object-position: center;
+  margin: 0 3px 40px;
```

you can use `auto` here
```suggestion
  margin: 0 auto 40px;
```